packages/worker/src/index.js

```diff
@@ -13,8 +14,11 @@ const redis = require("./utilities/redis")
 
 const app = new Koa()
 
+app.keys = ["secret", "key"]
```

Comment:
I don't think we need this `koaSession` stuff, because we have our own session management now in redis.